requirements.txt

```diff
@@ -5,7 +5,7 @@ humanfriendly==2.1
 invoke==0.13.0
 mako==1.0.1
 raven==5.27.0
-setuptools==30.4.0
+setuptools==37.0.0
```

Comment:
The ticket was targeting >=31.0. I tried the [latest](https://pypi.python.org/pypi/setuptools) version and nothing breaks. The doc is also updated.